.github/workflows/pr-lint.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check ticket name conforms to requirements
-        run: 'echo ${{ github.event.pull_request.title }} | grep -i -E -q "^((apm|niad)-[0-9]+): ([a-zA-Z]+)|(dependabot\/)"'
+        run: 'echo ${{ github.event.pull_request.title }} | grep -i -E -q "^((niad)-[0-9]+): ([a-z]+)|(dependabot\/)"'
 
   check-branch-name:
     runs-on: ubuntu-latest
     steps:
-      - name: Check branch name confirms to requirements
-        run: 'echo ${{ github.event.pull_request.head.ref }} | grep -i -E -q "(((feature|bug))/(niad)-[0-9]+-[a-zA-Z]+)"'
+      - name: Check branch name conforms to requirements
+        run: 'echo ${{ github.event.pull_request.head.ref }} | grep -i -E -q "(((feature|bug))/(niad)-[0-9]+-[a-z]+)"'
```

Comment:
We definitely have some extra round brackets which are not necessary

Reply:
@antonlebedjko i've made these changes and removed the unnecessary brackets